setup.py

```diff
@@ -66,6 +66,9 @@
     url="https://buildstream.build",
     project_urls={
         "Documentation": "https://apache.github.io/buildstream-plugins/",
+        "Source": "https://github.com/apache/buildstream-plugins/",
+        "Tracker": "https://github.com/apache/buildstream-plugins/",
```

Comment:
shouldn't this be https://github.com/apache/buildstream-plugins/issues ?

Reply:
you're right, i'll fix that shortly

thanks for reviewing this!